lib/LaTeXML/Core/State.pm

```diff
@@ -422,6 +422,26 @@ sub lookupExpandable {
     return $defn; }
   return; }
 
+# Whether token must be wrapped as dont_expand
+sub isDontExpandable {
```

Comment:
Maybe name it `can_expansion_be_deferred` ? It's hard to find a good name.

Or my first instinct was: this subroutine will only be used in one caller, can we merge it back with it? Namely, the `with_dont_expand` sub.

Reply:
Kinda liked the name, exactly because it is ridiculous; gotta have *some* fun. And liked it in `State` because it accesses so much internal stuff; and leaves `with_dont_expand` readable.